lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/dataaccess/DBOSubmissionDAOImpl.java

```diff
@@ -136,6 +139,38 @@
 	private static final String LIMIT = "LIMIT";
 	private static final String OFFSET = "OFFSET";
 
+	/*
+	select s.submission_serialized, ss.modified_on
+	from s, ss
+	where s.id=ss.s_id and
+	s.ar_id=? and
+	ss.state='approved' and
+	ss.modified_on = 
+	select(max(ss2.modified_on)
+	from s2, ss2
+	where 
+	s2.ar_id=s.ar_id and s2.rp_id=s.rp_id
+	and s2.id=ss2.s_id and ss2.state=approved
+	group by s2.ar_id, s2.rp_id)
+	order by ss.modified_on asc;
+	*/
```

Comment:
Thank you for the comment with the easy to read sql!  To comprehend, I only had to go to the real sql to figure out what the table aliases referenced.

Reply:
I just replaced the hand written pseudo-code with the actual generated SQL.